cpp/include/Ice/InputStream.h

```diff
@@ -591,7 +583,8 @@ namespace Ice
             }
         }
 
-#ifdef ICE_UNALIGNED // Optimization with unaligned reads
+#if defined(ICE_UNALIGNED) || (defined(_WIN32) && defined(ICE_API_EXPORTS))
```

Comment:
On Windows, we need to build & export these functions all the time, otherwise they aren't available when we "opt in" to get them.